QUICKSTART.md

```diff
@@ -0,0 +1,197 @@
+# Quick Start Guide
+
+A quick start guide to getting a system running with the latest TrenchBoot
+Secure Launch bits. Note that this is a bare bones document meant to help
+someone get up and running with Secure Launch. It does not contain detailed
+descriptions of all the technologies and terminology involved in doing a
+secure late launch. The repository this document resides in contains a plethora
+of other resources that can be used to understand things more broadly.
+
+## Platforms
+
+The current patchset (version 8) only supports Intel TXT. AMD SKINIT support
+is in the works and coming soon.
+
+An Intel system (desktop, server, laptop) needs to be a vPro SKU in order to
+have TXT support available. Generally speaking, vPro systems will advertise this
+with a sticker somewhere on the unit. Intel TXT support usually needs to be
+enabled in the firmware setup program. It depends on both the TPM and VTd being
+enabled. The details on how to do this are system specific. To see if the CPU
+supports TXT, run the following (SMX (Safe Mode Extensions) indicates the CPU
+does support TXT):
+
+`# grep smx /proc/cpuinfo`
+
+## Linux
+
+TrenchBoot and the Secure Launch feature are an active project that is in the
+process of being upstreamed to the Linux project.
```

Comment:
How about:

TrenchBoot is an active open-source project for system launch integrity, from which the Secure Launch feature is being upstreamed to the Linux kernel.

Reply:
Sure can do